### ISSUE TITLE
Albany build broken due to Kokkos::Experimental::Impl not being defined

### ISSUE THREAD

**Issue by ikalash** [gahansen/Albany#269]
It looks like Albany failed to build in the nightlies last night.  I can only see the external CDash, as I am on foreign travel:

https://my.cdash.org/viewBuildError.php?buildid=1381133

but I am guessing this happened in the internal tests too.  Here is the error cut/paste here:

```
repos/Albany/src/PHAL_Utilities_Def.hpp:224:39: error: 'Kokkos::Experimental::Impl' has not been declared
```

Can someone please look at what is going on / fix it while I am away?

**Comment by ibaned** [comment]
Should be as easy as removing the `Experimental::` part, but I'll start a build to test the fix too.

**Comment by ibaned** [comment]
Should be fixed in 0df223ffc . Thanks for pointing this out!
